layouts/partials/footer.html

```diff
@@ -12,6 +12,7 @@ <h5 class="text-white">Legal</h5>
 			<ul class="list-unstyled">
 				<!-- <li><a class="footer__link" href="#">Privacy Policy</a></li> -->
 				<li><a class="footer__link" href="/terms-of-service">Terms of Service</a></li>
+				<li><a class="footer__link" href="https://storj.io/privacy-policy/">Privacy Policy</a></li>
```

Comment:
Link points to the storj.io privacy policy.

Reply:
doing that for now, we're not sure if any terms need to be changed for the tardigrade specific one yet

Reply:
Ok, makes sense now.